bin/import-osm

```diff
@@ -14,29 +14,42 @@ export PGPORT="${POSTGRES_PORT:-${PGPORT:-5432}}"
 
 
 : "${DIFF_MODE:=true}"
+: "${APPEND_MODE:=false}"
 
 
 function import_pbf() {
     local pbf_file="$1"
-    local diff_flag=""
+
+    local extra_flags=""
+    local action_msg=""
+    local mode_msg=""
+
     if [ "$DIFF_MODE" = true ]; then
-        diff_flag="-diff"
-        echo "Importing $pbf_file into $PGHOST:$PGPORT/$PGDATABASE using Imposm in DIFF mode..."
+        extra_flags="$extra_flags -diff"
+        mode_msg="NORMAL"
+    else
+        mode_msg="DIFF"
+    fi
```

Comment:
> if/else would be a bit more readable:
>
> ```
>    local extra_flags=""
>    local action_msg=""
>    local mode_msg=""
>
>    if [ "$DIFF_MODE" = true ]; then
>        extra_flags="$extra_flags -diff"
>        mode_msg="NORMAL"
>    else
>        mode_msg="DIFF"
>    fi
>    
>    if [ "$APPEND_MODE" = true ]; then
>        extra_flags="$extra_flags -appendcache"
>        action_msg="Appending"
>    else
>        extra_flags="$extra_flags -overwritecache"
>        action_msg="Importing"
>    fi
>
>    echo "$action_msg $pbf_file into $PGHOST:$PGPORT/$PGDATABASE using Imposm in $mode_msg mode..."
>```
> and use the `$extra_flags` instead of `$cache_flag` below

_Originally posted by @nyurik in https://github.com/openmaptiles/openmaptiles-tools/pull/365#discussion_r709323888_

Sorry I didn' saw the mistake in your suggestion and copy & pasted it ...

Reply:
ah, great, so just a messaging error it seems :)  Yeah, turns out suggestions without testing the code are not as good as actually trying it out :)  Want to submit a patch?  I will release it as 6.0.1